httptransport/logginghandler.go

```diff
@@ -0,0 +1,34 @@
+package httptransport
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/rs/zerolog"
+)
+
+type httpStatusWriter struct {
+	http.ResponseWriter
+
+	StatusCode int
+}
+
+// LoggingHandler will log HTTP requests using the pre initialized zerolog.
+func LoggingHandler(next http.Handler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		start := time.Now()
+
+		log := zerolog.Ctx(r.Context())
+
+		next.ServeHTTP(w, r)
+
+		log.Info().
+			Str("remote addr", r.RemoteAddr).
+			Str("method", r.Method).
+			Str("request uri", r.RequestURI).
+			Str("status", strconv.Itoa(http.StatusOK)).
```

Comment:
Does this always logs 200?

Reply:
That's embarrassing *runs* I hate how that works. Readding